### DISCUSSION TITLE
Convert nested JSON into a tabular data package.

### DISCUSSION THREAD

**Discussion by kindly** [frictionlessdata/frictionlessdata.io#713] [Show and tell]
I have been working on a tool to convert deeply nested JSON data into tabular data called [flatterer](https://flatterer.opendata.coop/).

One of its outputs is a tabular data package.  This could be an interesting addition to project as more and more data from APIs are JSON and it is normally burdensome to manually convert them to tabular form for analysis or to use frictionless data tools on them.

Here is a example notebook of how the python API can be used:
https://deepnote.com/@david-raznick/Flatterer-Demo-data-package-e16vTBI7SXaSshHm_92o7w

It can also be used as a command line tool and there is a [web interface](https://flatterer.herokuapp.com/) with a datapackage style zip export.

**Comment by roll** [comment]
@kindly 
Nice work! cc @lwinfree @sapetti9

**Comment by sapetti9** [comment]
Very nice work @kindly! Would you be interested in coming and present it at our community call in April? It's on 28th at 3pm CET. We would also very happily publish a blog about it on the [project website](https://frictionlessdata.io/blog/), if you feel like writing one. Drop me and email to get more details at sara.petti[at]okfn.org.
